packages/contracts/.env.example

```diff
@@ -0,0 +1,10 @@
+# core
+DEFAULT_NETWORK=hardhat
```

Comment:
where's this used? iirc in hardhat the hardhat network is the default already

Reply:
this can be helpful for custom default networks (running nodes different from current hardhat environment / network).

Reply:
oh right thought it'd be more convenient to pass --network $NETWORK to the command so you don't have to switch env var but this works too